spec/javascripts/models/beta/iteration_spec.js

```diff
@@ -55,75 +82,327 @@ describe("iteration", function() {
-    beforeEach(function() {
-      this.project = {
+    describe("with empty array of stories", function() {
+      const project = {
         startDate: "2018-09-03T16:00:00",
         iterationLength: 1,
         defaultVelocity: 2
       };
-      this.initialSprintNumber = Iteration.getCurrentIteration(this.project);
-    });
+      const initialSprintNumber = Iteration.getCurrentIteration(project);
 
-    describe("with empty array of stories", function() {
       describe("when there are no past iterations", () => {
-        it("should return an empty array of sprints", function() {
+        it("return an empty array of sprints", () => {
           const stories = [];
           const sprints = Iteration.groupBySprints(
             stories,
-            this.project,
-            this.initialSprintNumber,
+            project,
+            initialSprintNumber,
             []
           );
           expect(sprints).toEqual([]);
         });
       });
+
       describe("when there are 3 past iterations", () => {
-        it("should return an empty array of sprints", function() {
+        it("return an empty array of sprints", () => {
           const stories = [];
           const sprints = Iteration.groupBySprints(
             stories,
-            this.project,
-            this.initialSprintNumber,
+            project,
+            initialSprintNumber,
             [{points: 3}, {points: 4}, {points: 5}]
           );
           expect(sprints).toEqual([]);
         });
       })
     });
 
-    describe("with 2 unstarted features with 1 point", function() {
-      describe("when there are no past iterations", () => {
-        it("should return an array with 1 item", function() {
+    describe('when default velocity is 2', () => {
+      beforeEach(function() {
+        this.project = {
+          startDate: "2018-09-03T16:00:00",
+          iterationLength: 1,
+          defaultVelocity: 2
+        };
+        this.initialSprintNumber = Iteration.getCurrentIteration(this.project);
+      });
+  
+      describe("with 2 unstarted features with 1 point", function() {
+        describe("when there are no past iterations", () => {
+          it("return an array with 1 item", function() {
+            const stories = [
+              {
+                id: 1,
+                position: "3.2",
+                state: "unstarted",
+                estimate: 1,
+                storyType: "feature"
+              },
+              {
+                id: 2,
+                position: "10",
+                state: "unstarted",
+                estimate: 1,
+                storyType: "feature"
+              }
+            ];
+  
+            const sprints = Iteration.groupBySprints(
+              stories,
+              this.project,
+              this.initialSprintNumber,
+              []
+            );
+  
+            expect(sprints.length).toEqual(1);
+            expect(sprints[0].stories.length).toEqual(2);
```

Comment:
This line is not described in the test title, can we add it to the title as well?